things/api.py

```diff
@@ -530,6 +530,14 @@ def deadlines(**kwargs):
 
 # Interact with Things app
 
+def token(**kwargs) -> str:
+    """
+    Returns the API token.
```

Comment:
@AlexanderWillner How about linking to the documentation, e.g. here https://culturedcode.com/things/support/articles/2803573/ to explain what one can use this token for?